addons/mail/static/tests/discuss/discuss_tests.js

```diff
@@ -61,6 +61,34 @@ QUnit.module("mail", (hooks) => {
         );
     });
 
+    QUnit.test("can change the thread name of #general", async (assert) => {
+        const server = new TestServer();
+        server.addChannel(1, "general", "General announcements...");
+        const env = makeTestEnv((route, params) => {
+            if (route === "/web/dataset/call_kw/mail.channel/channel_rename") {
+                assert.step(route);
+            }
+            return server.rpc(route, params);
+        });
+        await mount(Discuss, target, { env });
+        env.services["mail.messaging"].setDiscussThread(1);
+
+        assert.containsOnce(target, ".o-mail-discuss-thread-name");
+        const threadNameElement = target.querySelector(".o-mail-discuss-thread-name");
+
+        await click(threadNameElement);
+        assert.strictEqual(threadNameElement.value, "general");
+        await editInput(target, ".o-mail-discuss-thread-name", "special");
+        await triggerEvent(target, ".o-mail-discuss-thread-name", "keydown", {
+            key: "Enter",
+        });
+        assert.strictEqual(threadNameElement.value, "special");
+
+        assert.verifySteps([
+            "/web/dataset/call_kw/mail.channel/channel_rename",
+        ]);
```

Comment:
Prettier would keep everything in a single line

Reply:
for me, it will format the code like this.